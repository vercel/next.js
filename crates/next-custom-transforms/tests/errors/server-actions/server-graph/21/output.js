'use server';
export async function fn() {}
