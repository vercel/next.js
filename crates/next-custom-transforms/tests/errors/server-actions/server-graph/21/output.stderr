  x Conflicting directives "use server" and "use cache" found in the same file. You cannot place both directives at the top of a file. Please remove one of them.
  | 
   ,-[input.js:2:1]
 1 | 'use cache'
 2 | 'use server'
   : ^^^^^^^^^^^^
   `----
